travis-ymls/concat-stream.travis.yml

```diff
@@ -0,0 +1,20 @@
+# ----------------------------------------------------------------------------
+#
+# Package             : concat-stream 
+# Source Repo         : https://github.com/maxogden/concat-stream.git
+# Travis Job Link     : https://github.com/nagesh4193/concat-stream
```

Comment:
We need the link of the travis job here not the GitHub fork

Reply:
I have updated the PR with the required Travis job link.